src/current/cockroachcloud/cluster-management.md

```diff
@@ -5,6 +5,10 @@ toc: true
 docs_area: manage
 ---
 
+{{site.data.alerts.callout_info}}
+CockroachDB Standard, our new, enterprise-ready plan, is currently in [Preview]({% link {{ site.current_cloud_version }}/cockroachdb-feature-availability.md %}).
```

Comment:
As a followup, it would be good to move this to an include, and to use the variable for Standard. If you leave the admonition wrapper out of the include, you can even use it in the feature-availability page itself.

Reply:
Thanks. Agreed.